build.py

```diff
@@ -1189,7 +1189,7 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu,
  && rm -f ${_CUDA_COMPAT_PATH}/lib
 '''
     else:
-        add_cpu_libs_to_linux_dockerfile(df, backends, target_machine)
+        df += add_cpu_libs_to_linux_dockerfile(df, backends, target_machine)
```

Comment:
I think this would add the libraries twice.

Reply:
Thanks for raising this point It looks like strings don't get modified if passed as an argument: https://www.geeksforgeeks.org/is-python-call-by-reference-or-call-by-value/#

That's why the libraries weren't actually getting added in the CI. Removed that parameter to make it clearer and faster. Let me know if that works for you.